test/pytest/test_extensions.py

```diff
@@ -74,8 +97,6 @@ def format(self, node):
     data_T input[CONFIG_T::n_in],
     data_T reversed[CONFIG_T::n_in]
 ) {
-    #pragma HLS PIPELINE
```

Comment:
Does this line cause problems for Intel HLS compiler? What if we wrap it in `#ifdef __SYNTHESIS__` and do similar for Intel compiler

Reply:
I think it won't (I think there is a flag for ignoring unknown pragmas), but PyTests do not run HLS synthesis anyway, so this line does not really do anything. Only reason I removed it is to make it "technically" correct.

Reply:
That entire file used to be an example of extensions API that also served as a test, so that block is an example of (Vivado) HLS code that users should write themselves. I don't see how distilling it to pure C++ code makes it "tecnhically" correct ;-) But it doesn't matter, since now it is a test that serves as a (slightly convoluted) example.